st2common/st2common/models/api/action.py

```diff
@@ -497,6 +497,27 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "array",
                 "items": {"type": "string"},
                 "description": "Possible parameter format."
+            },
+            "aliases": {
```

Comment:
We discussed it in the chatops meeting on Mon. It's pretty much same as formats (matching user strings), with the notable exception of not being exposed in Hubot's `!help`. It's part of my effort to "humanize" the bot: `formats` contain well-defined string commans, and `aliases` could contain regexes for the same command, or alternative strings, catching common typos or even adding a tiny bit of NLP. 

This came from experience in exposing new people to chatops: if you have a lot of aliases, people will forget exact commands, or misspell them, or try to use a different syntax, so it's nice to actually have some flexibility on that without flooding `!help` with 10 variations on a command or a huge regex.

So the name pretty much sums it up: it's a list of aliases for format.